src/RegexProvider/RegexProvider.fs

```diff
@@ -11,10 +11,20 @@ module internal TypedRegex =
     let typedRegex() = 
         let regexType = erasedType<Regex> thisAssembly rootNamespace "Regex"
         regexType.DefineStaticParameters(
-            parameters = [ProvidedStaticParameter("pattern", typeof<string>)], 
+            parameters =
+                [
+                    ProvidedStaticParameter("pattern", typeof<string>)
+                    ProvidedStaticParameter("generatedMethodsPrefix", typeof<string>, "")
```

Comment:
If you don't mind breaking compatibility by default, of course this is much more convenient